### PR TITLE
Make currency a required key

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
@ScottDowne r?

File: server.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ module.exports = function(options) {
           schema: {
             id: Joi.string(),
             frequency: Joi.string().valid("monthly", "one-time"),
-            currency: Joi.string().min(3).max(3),
+            currency: Joi.any().valid(currencyFor.stripe).required(),
```
Comment (alicoding) [inline server.js]: Why change this one too?

File: server.js [hunk 2]
```diff
@@ -158,7 +158,7 @@ module.exports = function(options) {
...
```
Comment (alicoding) [inline server.js]: and this

File: server.js (hunk 1 above)
Comment (ScottDowne) [inline server.js]: I think at this point, this is just covering ourselves to not break what we pass back.

**Comment by ScottDowne** [comment]
@jbuck is there a test for a paypabl disabled currency being used in paypal?

**Comment by jbuck** [comment]
@ScottDowne are you referring to this test? https://github.com/mozilla/donate.mozilla.org/pull/1113/files#diff-0ffce28eb3811edea299bff9abe7ca50R27

**Comment by jbuck** [comment]
@ScottDowne Added the test!

**Comment by jbuck** [comment]
@alicoding yeah, I'm trying to be consistent about our validations; I think it's something we can gradually improve though.

**Pull request merged**